### PR TITLE
[DC-815] [DC-1205] Update Drug Supply & Refill CR to New Base Class & Correct Pipeline Step

### PR DESCRIPTION
**Pull request by hiro-mishima**

### CODE REVIEW

File: data_steward/cdr_cleaner/cleaning_rules/drug_refills_days_supply.py [hunk 1]
```diff
@@ -1,68 +1,134 @@
...
+                 sandbox_dataset_id,
+                 table_namer=None):
+        """
+        Initialize the class with proper information.
+        """
+        desc = ('Ensure drug refills < 10 and days_supply < 180.')
+        super().__init__(issue_numbers=['DC403', 'DC815'],
+                         description=desc,
+                         affected_datasets=[cdr_consts.COMBINED],
+                         affected_tables=DRUG_EXPOSURE,
```
Comment (lrwb-aou) [inline data_steward/cdr_cleaner/cleaning_rules/drug_refills_days_supply.py:51]: This needs to be a list.
```suggestion
                         affected_tables=[DRUG_EXPOSURE],
```

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/drug_refills_days_supply_test.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                                                  cls.dataset_id,
+                                                  cls.sandbox_id)
+
+        cls.fq_table_names = [
+            f'{cls.project_id}.{cls.dataset_id}.{DRUG_EXPOSURE}'
+        ]
+
+        cls.fq_sandbox_table_names = [
+            f'{cls.project_id}.{cls.sandbox_id}.{cls.rule_instance.sandbox_table_for(DRUG_EXPOSURE)}'
+        ]
```
Comment (lrwb-aou) [inline tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/drug_refills_days_supply_test.py:41]: This works, but doesn't cause the error to surface with the `affected_tables` variable when that isn't set correctly.  You can accomplish the same thing with the following and you don't have to "tweak" this code as often.  This also lets you the import of `DRUG_EXPOSURE`.
```suggestion
        cls.fq_sandbox_table_names = []
        for table in cls.rule_instance.get_sandbox_tablenames():
            cls.fq_sandbox_table_names.append(f'{cls.project_id}.{cls.sandbox_id}.{table}')
```

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/drug_refills_days_supply_test.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+Original Issues: DC-403, DC-815
+"""
+# Python Imports
+import os
+
+# Project imports
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.drug_refills_days_supply import DrugRefillsDaysSupply
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+from common import DRUG_EXPOSURE, JINJA_ENV
```
Comment (lrwb-aou) [inline tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/drug_refills_days_supply_test.py:12]: You do not need to import `JINJA_ENV`.  You can inherit this from the base test class by using its `jinja_env` variable.  So, line 50 could be
```python
        INSERT_DRUG_EXPOSURE_QUERY = self.jinja_env.from_string("""
```

**Review by jp3477 — APPROVED** [review]
Approved, nothing to add to Lauren's suggestions

**Pull request merged**